packages/client/src/v2-events/utils/pdf/index.ts

```diff
@@ -0,0 +1,232 @@
+/*
```

Comment:
Let's move these files from v2-events/utils to `v2-events/features/events/actions/print-certificate/pdf` for instance as these are directly related to printing a certificate